docs/ui/components/CommandMenu/utils.ts

```diff
@@ -0,0 +1,141 @@
+import type { Dispatch, SetStateAction } from 'react';
+
+import type { AlgoliaItemHierarchy, AlgoliaItemType } from './types';
+
+export const getItems = async <T>(
```

Comment:
nitpick: in our other codebases, we'd name this `getItemsAsync`, so that it's clear it's an async function when it's called.

Reply:
We could make this method non async easily, but I just went and renamed them for now.